mac

```diff
@@ -146,12 +150,17 @@ cask "lastpass"
 cask "insomnia"
 cask "flycut"
 cask "evernote"
+cask "bitwarden"
+cask "caffeine"
```

Comment:
What's caffeine?

Reply:
Ultra-tiny app that puts a coffee cup on the system control bar at top of screen. Can click the cup to keep your computer from snoozing. Great when you've got something long-running going in the terminal (like a scraper) and want to walk away and make lunch.